project_scrum/security/res_groups.xml

```diff
@@ -0,0 +1,15 @@
+<odoo>
```

Comment:
Missing: <?xml version="1.0" encoding="utf-8"?>

(Also missing from other views, please add them to all xml documents).

Reply:
Sorry forgot to paste: <?xml version="1.0" encoding="utf-8"?>

Reply:
No, it is an error in github comments. retry: ```<?xml version="1.0" encoding="utf-8"?>```